lms/djangoapps/instructor/enrollment.py

```diff
@@ -526,11 +526,19 @@ def send_mail_to_student(student, param_dict, language=None):
         param_dict['course_name'] = parent_course_name if is_ccx else Text(course.display_name_with_default)
         param_dict['class_name'] = Text(course.display_name_with_default) if is_ccx else None
 
-    param_dict['site_name'] = configuration_helpers.get_value(
+    site_name = configuration_helpers.get_value(
```

Comment:
I don't think it's necessary to change the variable just to assign it again in line 540, you can use whatever value `param_dict['site_name']` has. @sergivalero20